src/ResourceManagement/DataLake.Store/DataLakeStore.Tests/DataTransferTests/DataLakeStoreClientTests.cs

```diff
@@ -80,33 +81,27 @@ public void DataLakeUploader_InvalidParameters()
             Assert.Throws<FileNotFoundException>(
                 () =>
                 {
-                    new DataLakeStoreUploader(new UploadParameters(invalidFilePath, "1", "foo", maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd());
+                    new DataLakeStoreTransferClient(new TransferParameters(invalidFilePath, "1", "foo", maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd());
                 });
 
             //no target stream
             Assert.Throws<ArgumentNullException>(
-                () => { new DataLakeStoreUploader(new UploadParameters(_largeFilePath, null, "foo", maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
+                () => { new DataLakeStoreTransferClient(new TransferParameters(_largeFilePath, null, "foo", maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
 
             //target stream ends in '/'
             Assert.Throws<ArgumentException>(
-                () => { new DataLakeStoreUploader(new UploadParameters(_largeFilePath, "1/", "foo", maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
+                () => { new DataLakeStoreTransferClient(new TransferParameters(_largeFilePath, "1/", "foo", maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
 
             //no account name
             Assert.Throws<ArgumentNullException>(
-                () => { new DataLakeStoreUploader(new UploadParameters(_largeFilePath, "1", null, maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
+                () => { new DataLakeStoreTransferClient(new TransferParameters(_largeFilePath, "1", null, maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
 
             //bad thread count
             Assert.Throws<ArgumentOutOfRangeException>(
-                () => { new DataLakeStoreUploader(new UploadParameters(_largeFilePath, "1", "foo", perFileThreadCount: 0, maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
+                () => { new DataLakeStoreTransferClient(new TransferParameters(_largeFilePath, "1", "foo", perFileThreadCount: DataLakeStoreTransferClient.MaxAllowedThreadsPerFile + 1, maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
 
             Assert.Throws<ArgumentOutOfRangeException>(
-                () => { new DataLakeStoreUploader(new UploadParameters(_largeFilePath, "1", "foo", perFileThreadCount: DataLakeStoreUploader.MaxAllowedThreads + 1, maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
-
-            Assert.Throws<ArgumentOutOfRangeException>(
-                () => { new DataLakeStoreUploader(new UploadParameters(_largeFilePath, "1", "foo", concurrentFileCount: 0, maxSegmentLength: 4 * 1024 * 1024), new InMemoryFrontEnd()); });
```

Comment:
Because they are tested elsewhere now. <= 0 thread counts now result in a default thread count being computed. Previously it would result in an error, so those tests are no longer valid and the new tests should validate this.